docker-compose.yml

```diff
@@ -30,6 +30,9 @@ services:
       DB_HOST: postgres
       DB_PORT: 5432
       START_BLOCK: 1
+      ## Set the below two values to run all migration required from version `PREVIOUS_SQ_VERSION` to `CURRENT_SQ_VERSION`
+      PREVIOUS_SQ_VERSION: 5.3.8
+      CURRENT_SQ_VERSION: 5.4.3
```

Comment:
```suggestion
```

If we can't use an ORM migration tool, then we should be able to read this somehow.

Reply:
I think `subqueries` table might record the version of SQ running, but its empty when I look at polymesh-local table.

If not we should probably mark a table with that information somehow